vaadin-list-mixin.html

```diff
@@ -240,12 +291,19 @@
       firstItem && firstItem.focus();
     }
 
-    /* @protected */
+    /**
+     * @protected
+     * @return {HTMLElement}
```

Comment:
It looks like `_scrollerElement` is assumed to not be null in few places where it is used so this should probably be:
```suggestion
     * @return {HTMLElement!}
     * @protected
```
to get:
```ts
readonly _scrollerElement: HTMLElement;
```
without `| null`

Nit: also make `@protected` the last annotation for consistency.

Reply:
Done.